src/Compatibility/Core/src/Android/Renderers/ListViewAdapter.cs

```diff
@@ -64,7 +65,7 @@ public ListViewAdapter(Context context, AListView realListView, ListView listVie
 			realListView.OnItemClickListener = this;
 			realListView.OnItemLongClickListener = this;
 
-			MessagingCenter.Subscribe<ListViewAdapter>(this, Platform.CloseContextActionsSignalName, lva => CloseContextActions());
+			WeakReferenceMessenger.Default.Register<CloseContextActionsMessage>(this, (recipient, msg) => CloseContextActions());
```

Comment:
This is introducing a closure, we can skip the allocation by invoking directly on the recipient:

```suggestion
			WeakReferenceMessenger.Default.Register<ListViewAdapter, CloseContextActionsMessage>(this, (r, _) => r.CloseContextActions());
```